src/core/lib/gpr/sync_abseil.cc

```diff
@@ -62,7 +62,7 @@ void gpr_mu_unlock(gpr_mu* mu) ABSL_NO_THREAD_SAFETY_ANALYSIS {
 
 int gpr_mu_trylock(gpr_mu* mu) {
   GPR_TIMER_SCOPE("gpr_mu_trylock", 0);
-  int ret = reinterpret_cast<absl::Mutex*>(mu)->TryLock() == true;
+  int ret = reinterpret_cast<absl::Mutex*>(mu)->TryLock();
   return ret;
```

Comment:
I think this could be simplified to a one-liner.

Reply:
done.